packages/runtime/native/shared/Declarations.ml

```diff
@@ -446,6 +446,16 @@ let overflows x =
 let overflowX x = Rule.declaration ({js|overflowX|js}, Overflow.toString x)
 let overflowY x = Rule.declaration ({js|overflowY|js}, Overflow.toString x)
 
+let overflowClipMargin x =
+  Rule.declaration ({js|overflowClipMargin|js}, OverflowClipMargin.toString x)
+
+let overflowClipMargin2 ?(clipEdgeOrigin = `paddingBox) ?(margin = `px 0) () =
```

Comment:
can we keep margin not being a labelled argument? (just for cuteness)

Reply:
yeah better remove the dangling unit